lib/pengine/native.c

```diff
@@ -312,48 +312,28 @@ native_find_rsc(pe_resource_t * rsc, const char *id, const pe_node_t *on_node,
     return NULL;
 }
 
+// create is ignored
 char *
```

Comment:
That is still building with warnings as errors?

Reply:
Yes, function arguments don't need to be marked as unused, just declared variables. We have tons of functions that ignore arguments, mostly callbacks or method implementations that need a certain signature but don't actually need the info.